chatmd/util.py

```
@@ -94,16 +97,69 @@ def set_cached_folder_id(name: str, folder_id: str) -> None:
             runs = json.loads(RUNS_PATH.read_text(encoding="utf-8"))
             if not isinstance(runs, list):
                 runs = []
         else:
             runs = []
-        runs.append(record)
+        runs.append(payload)
         # Keep last 200
         runs = runs[-200:]
         RUNS_PATH.write_text(json.dumps(runs, indent=2), encoding="utf-8")
     except Exception:
         pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)